pack/ptc/dca_encounter.json

```diff
@@ -376,60 +376,70 @@
         "traits": "Ancient One. Elite.",
         "type_code": "enemy"
     },
     {
         "code": "03333",
         "encounter_code": "dim_carcosa",
+        "encounter_position": 24,
         "enemy_evade": 3,
         "enemy_fight": 3,
         "enemy_horror": 2,
         "faction_code": "mythos",
         "health": 9,
+        "health_per_investigator": true,
         "illustrator": "Stephen Somers",
-        "name": "Hastur",
+        "is_unique": true,
+        "name": "Hastur",
         "pack_code": "dca",
         "position": 333,
         "quantity": 1,
         "subname": "Lord of Carcosa",
-        "text": "<b>Prey</b> - Most remaining sanity.<br/>Hunter. Massive.<br/>If Hastur's attack deals you horror in excess of your sanity, you take 1 damage, as well.<br/>While Hastur is being attacked or evaded by an investigator with no remaining sanity, treat each [elder_sign], \"+1,\" \"0,\" or \"-1\" token that investigator reveals as a [auto-fail] token.",
+        "text": "<b>Prey</b> - Most remaining sanity.\nHunter. Massive.\nIf Hastur's attack deals you horror in excess of your sanity, you take 1 damage, as well.\nWhile Hastur is being attacked or evaded by an investigator with no remaining sanity, treat each [elder_sign], \"+1,\" \"0,\" or \"-1\" token that investigator reveals as a [auto-fail] token.",
         "traits": "Ancient One. Elite.",
         "type_code": "enemy"
     },
     {
         "code": "03334",
         "encounter_code": "dim_carcosa",
+        "encounter_position": 25,
         "enemy_evade": 2,
         "enemy_fight": 3,
         "enemy_horror": 4,
         "faction_code": "mythos",
         "health": 8,
+        "health_per_investigator": true,
         "illustrator": "Stephen Somers",
-        "name": "Hastur",
+        "is_unique": true,
+        "name": "Hastur",
         "pack_code": "dca",
         "position": 334,
         "quantity": 1,
         "subname": "The Tattered King",
-        "text": "<b>Prey</b> - Most remaining sanity.<br/>Hunter.<br/>If Hastur's attack deals you horror in excess of your sanity, you take 1 damage, as well.<br/>While Hastur is being attacked or evaded by an investigator with no remaining sanity, treat each [elder_sign], \"+1,\" \"0,\" or \"-1\" token that investigator reveals as a [auto-fail] token.",
+        "text": "<b>Prey</b> - Most remaining sanity.\nHunter.\nIf Hastur's attack deals you horror in excess of your sanity, you take 1 damage, as well.\nWhile Hastur is being attacked or evaded by an investigator with no remaining sanity, treat each [elder_sign], \"+1,\" \"0,\" or \"-1\" token that investigator reveals as a [auto-fail] token.",
```

Comment:
Please change [auto-fail] to [auto_fail]. (For future reference: the data from cardgamedb uses [per-investigator], [elder-sign] and [auto-fail] and arkhamdb uses [per_investigator], [elder_sign] and [auto_fail].)